dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/identifiable/IdentifiableRepositoryImpl.java

```diff
@@ -367,42 +367,28 @@ public boolean deleteIdentifiers(UUID identifiableUuid) {
 
   @Override
   public SearchPageResponse<I> find(SearchPageRequest searchPageRequest) {
-    String commonSql =
-        " FROM "
-            + tableName
-            + " AS "
-            + tableAlias
-            + " LEFT JOIN LATERAL jsonb_object_keys("
-            + tableAlias
-            + ".label) lbl(keys) ON "
-            + tableAlias
-            + ".label IS NOT NULL"
-            + " LEFT JOIN LATERAL jsonb_object_keys("
-            + tableAlias
-            + ".description) dsc(keys) ON "
-            + tableAlias
-            + ".description IS NOT NULL";
-
+    String commonSql = " FROM " + tableName + " AS " + tableAlias;
     String searchTerm = searchPageRequest.getQuery();
     if (!StringUtils.hasText(searchTerm)) {
       return find(searchPageRequest, commonSql, Collections.EMPTY_MAP);
     }
 
     commonSql +=
         " WHERE ("
+            + "jsonb_path_exists("
             + tableAlias
-            + ".label->>lbl.keys ILIKE '%' || :searchTerm || '%'"
+            + ".label, ('$.* ? (@ like_regex \"' || :searchTerm || '\" flag \"iq\")')::jsonpath)"
```

Comment:
Glaube hier würde ein kleiner Kommentar was da passiert nicht schaden :-)

Reply:
Ja, zumindest einmal sollte da was erläutert werden.

Reply:
Ich würde das statt dem FYI ganz oben tatsächlich expliziter an der Stelle beschreiben was in Zeile 386 bezweckt wird, die String Concatenation und den Cast musst du glaub nicht erklären, das ist ja geläufig, dachte eher an das `'$.* ? (@ like ...)'`

Also sowas wie:

```
// Finds (case insensitively) labels that contain the search term, see `like_regex` example in
// https://www.postgresql.org/docs/12/functions-json.html#FUNCTIONS-SQLJSON-PATH